OrbitGl/TextRenderer.cpp

```diff
@@ -82,15 +81,22 @@ void TextRenderer::Init() {
   m_Initialized = true;
 }
 
-void TextRenderer::SetFontSize(int size) {
-  texture_font_t* font = m_FontsBySize[size];
-  if (font) {
-    m_Font = font;
-    current_font_size_ = size;
+void TextRenderer::SetFontSize(uint32_t size) {
+  if (!m_FontsBySize.count(size)) {
+    auto iterator_next = m_FontsBySize.upper_bound(size);
+    // If there isn't that font_size in the map, we will search for the next one or previous one
+    if (iterator_next != m_FontsBySize.end()) {
+      size = iterator_next->first;
+    } else if (iterator_next != m_FontsBySize.begin()) {
+      size = (--iterator_next)->first;
+    }
```

Comment:
Do we need an error-handling case here for `m_FontsBySize.begin() == m_FontsBySize.end()` (for the empty map)?

Reply:
I don't think so. We are filling the array at the first and we are not touching it anymore. But a check doesn't hurt.